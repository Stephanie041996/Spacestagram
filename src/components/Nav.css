nav {
    display: flex;
    width: 100%;
    height: 80px;
    background-color:#f9f9f9;
    align-items: center;
    justify-content: space-around;
  }
  
  .list-item,
  a {
    color: black;
  }
  
  nav a {
    color: black;
    text-decoration: none;
  }

  h4 {
    color: black;
  }
  