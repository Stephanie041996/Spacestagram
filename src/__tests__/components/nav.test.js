import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import Nav from '../../components/nav';
import store from '../../Store';

afterEach(cleanup);

it('should render the navbar', () => {
  const { asFragment } = render(
    <Provider store={store}>
      <BrowserRouter>
        <Nav />
      </BrowserRouter>
    </Provider>,
  );
  expect(
    asFragment(
      <Provider store={store}>
        <Nav />
      </Provider>,
    ),
  ).toMatchSnapshot();
});
