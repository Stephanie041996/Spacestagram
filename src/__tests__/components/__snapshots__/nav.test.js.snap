// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the navbar 1`] = `
<DocumentFragment>
  <nav>
    <a
      href="#top"
    >
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-bars fa-w-14 "
        data-icon="bars"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
          fill="currentColor"
        />
      </svg>
    </a>
    <h4>
      SPACESTAGRAM
    </h4>
    <a
      href="#top"
    >
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-undo fa-w-16 "
        data-icon="undo"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M212.333 224.333H12c-6.627 0-12-5.373-12-12V12C0 5.373 5.373 0 12 0h48c6.627 0 12 5.373 12 12v78.112C117.773 39.279 184.26 7.47 258.175 8.007c136.906.994 246.448 111.623 246.157 248.532C504.041 393.258 393.12 504 256.333 504c-64.089 0-122.496-24.313-166.51-64.215-5.099-4.622-5.334-12.554-.467-17.42l33.967-33.967c4.474-4.474 11.662-4.717 16.401-.525C170.76 415.336 211.58 432 256.333 432c97.268 0 176-78.716 176-176 0-97.267-78.716-176-176-176-58.496 0-110.28 28.476-142.274 72.333h98.274c6.627 0 12 5.373 12 12v48c0 6.627-5.373 12-12 12z"
          fill="currentColor"
        />
      </svg>
    </a>
  </nav>
</DocumentFragment>
`;
